about.html

```diff
@@ -55,10 +55,10 @@ <h3 class="title">
         <div class="col col-lg-6">
           <div class="description">
             <h2>
-              Contributions are valued the same way
+              Community contributions
             </h2>
             <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sit amet imperdiet felis. Vestibulum lorem justo, placerat ut placerat ut, pharetra eget lorem. Sed eu nisl vitae lacus molestie eleifend at accumsan ligula.
+              Vowpal Wabbit revolves around open community collaboration and research that empowers testing and exploration of real data for meaningful outcomes.

```

Comment:
What is this supposed to mean or cover?
What this discussed with John?

Reply:
The idea and context were already there, I just cut, rewrote and added here for potential. 

Made the edits to discuss with John...